libmscore/lyrics.cpp

```diff
@@ -110,6 +110,13 @@ void Lyrics::read(XmlReader& e)
             if (!readProperties(e))
                   e.unknown();
             }
+      if (!isStyled(Pid::OFFSET) && !e.pasteMode()) {
+            // fix offset for pre-3.1 scores
+            // 3.0: y offset was meaningless if autoplace is set
+            QPointF off = propertyDefault(Pid::OFFSET).toPointF();
```

Comment:
A minor point, but this `propertyDefault()` call can be moved inside the `if` block, otherwise the result is anyway not used.